### ISSUE TITLE
spacy-nightly 3.0.0rc2 does not provide the extra 'cuda110'

### ISSUE THREAD

**Issue by raffaem** [explosion/spaCy#6494]
## How to reproduce the problem

Installation instructions taken from [here](https://nightly.spacy.io/usage)
```
$pip install -U spacy-nightly[cuda110,transformers,lookups] --pre --user
```

```bash
Collecting spacy-nightly[cuda110,lookups,transformers]
  Downloading spacy_nightly-3.0.0rc2-cp38-cp38-win_amd64.whl (10.3 MB)
     |████████████████████████████████| 10.3 MB 1.3 MB/s
WARNING: spacy-nightly 3.0.0rc2 does not provide the extra 'cuda110'
```

## Your Environment

* Operating System: Windows 10
* Python Version Used: 3.8
* spaCy Version Used: 3 RC2
* Environment Information:

**Comment by adrianeboyd** [comment]
We'll definitely add it in the future, but in the meanwhile, you can just install the right version of `cupy-cudaNNN` separately.

**Comment by raffaem** [comment]
Still not fixed ...

Do I have to run

`pip install cupy-cuda110` ?

Do I also need the CUDA libraries on top of that?

**Comment by raffaem** [comment]
It also gives me the following error when I try to install the model:

```
$ python -m spacy download en_core_web_sm
⚠ Skipping pipeline package dependencies and setting `--no-deps`. You
don't seem to have the spaCy package itself installed (maybe because you've
built from source?), so installing the package dependencies would cause spaCy to
be downloaded, which probably isn't what you want. If the pipeline package has
other dependencies, you'll have to install them manually.
```

**Comment by adrianeboyd** [comment]
Yes, you need to install both CUDA and `cupy-cudaNNN` separately. It doesn't matter whether you install `spacy[cuda110]` or `spacy cupy-cuda110`.

The `--no-deps` error with `spacy-nightly` was fixed in #6502 and `cuda110` was added in #6297.
